### DISCUSSION TITLE
网站好像崩了

### DISCUSSION THREAD

**Discussion by erhulee** [alibaba/designable#401] [General]
![img_v3_02hf_e3c30202-8eb6-4ab7-9957-d5dd4dd8c8bg](https://github.com/user-attachments/assets/21ed4525-62a9-4ba0-b11f-c8f1aeadb9b3)

**Comment by aceHubert** [comment]
因为 react 发布了19，没有 umd 文件了，引用没有指定版本跳转到最新版本资源没了
